BusinessLogic/src/main/java/BusinessLogicLayer/EnronSearchEngine.java

```diff
@@ -58,6 +69,51 @@ public static void main(String[] args) throws Exception {
         shutdownAndAwaitTermination(pool);
     }
 
+    private static boolean checkTables() {
+        String sqlCheck = "SHOW TABLES";
+        try (Connection connection = Database.getConnection()) {
+            PreparedStatement ps = connection.prepareStatement(sqlCheck);
+            ResultSet res = ps.executeQuery();
```

Comment:
Both Prepared Statment and Result Set should also be included within the parameters of the try with resources block.